x-pack/legacy/plugins/maps/public/shared/layers/joins/left_inner_join.js

```diff
@@ -38,7 +34,7 @@ export class LeftInnerJoin {
   }
 
   getSourceId() {
-    return LeftInnerJoin.toHash(this._descriptor);
```

Comment:
could you add comment as to why this is?

It might still be useful to rename the method. The join-config can change and _not_ require a refresh (if the left-field configuration changes). So something like `getRightSourceId`.

Reply:
I will add a comment.

I don't think it makes sense to rename the method to `getRightSourceId`. The fact that `getSourceId` contains the id of the right source is an implementation detail of `LeftInnerJoin` that `VectorLayer` does not need to care about. `VectorLayer`'s only concern is that the id is consistent for a join.